packages/cerebral-demo/src/components/App/index.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-script-url */
+import React from 'react'
+import {connect} from 'cerebral/react'
+import LangSelector from '../LangSelector'
+import Login from '../Login'
+import Menu from '../Menu'
+import Timer from '../Timer'
+import Workspace from '../Workspace'
+import translations from '../../computed/translations'
+
+import './style.scss'
+
+const TaglineRe = /^(.*)\[Cerebral\](.*)$/
+
+export default connect(
+  { t: translations,
```

Comment:
In standard, if you put the opening brace up, the function arguments end up without any indentation which I find confusing:

```js
export default connect({
  t: translations,
  loggedIn: 'user.$loggedIn'
},
function Demo ({t, loggedIn}) {
  // ...
})
```

But I don't mind choosing this style if this better reflects usage.